### DISCUSSION TITLE
I am on 1.0.0, the UI is too big n weird on 1.3.0

### DISCUSSION THREAD

**Discussion by jekv2** [FossifyOrg/General-Discussion#539] [General]
I am on 1.0.0, the UI is too big n weird on 1.3.0 with answering and dialing.

please offer compact style like 1.0.0.
![Screenshot_20241220-125122_1](https://github.com/user-attachments/assets/3489dbd1-0287-4be7-a622-3036f4a4bfbc)

**Comment by Aga-C** [comment]
Is this screenshot from 1.0.0 or 1.3.0? What exactly is different?
